apmpackage/apm/agent/input/template.yml.hbs

```diff
@@ -13,6 +13,10 @@ apm-server:
     read_timeout: {{read_timeout}}
     response_headers: {{response_headers}}
     rum:
+        enabled: {{enable_rum}}
+        {{#if sourcemap_api_key}}
+        source_mapping.elasticsearch.api_key: {{sourcemap_api_key}}
+        {{/if}}
```

Comment:
```suggestion
```

And remove the `sourcemap_api_key` var from manifest.yml. We won't need this after introducing the Fleet method.